contributors/devel/cherry-picks.md

```diff
@@ -19,26 +19,46 @@ task is for backporting PRs from master to release branches.
+   * You will need to run the cherry-pick script separately for each patch release you want to cherry-pick to.
+
+ * Your cherry-pick PR targeted to the release branch will immediately get the
+   `do-not-merge/cherry-pick-not-approved` label. The [Patch Release Manager](https://git.k8s.io/sig-release/release-team/README.md#patch-release-manager)
    will triage PRs targeted to the branch.  Normal rules apply for code merge.
    * Reviewers `/lgtm` and owners `/approve` as they deem appropriate.
-   * The approving release branch owner is responsible for applying the
-     `cherrypick-approved` label.
+   * Milestones on cherry-pick PRs should be the milestone for the target release branch (for example, milestone 1.11 for a cherry-pick onto release-1.11).
+   * The [Patch Release Manager](https://git.k8s.io/sig-release/release-team/README.md#patch-release-manager) for the target release branch is responsible for
+   applying the `cherry-pick-approved` label.
+   You may cc them with `<@githubusername>` on your cherry-pick PR.
+
+## Cherry-pick Review
+
+Cherry-pick pull requests have an additional requirement compared to normal pull
+requests.
+They must be approved specifically for cherry-pick by Approvers.
+Additionally, the [Patch Release Manager](https://git.k8s.io/sig-release/release-team/README.md#patch-release-manager)
+is the final and sole authority on removing the `do-not-merge/cherry-pick-not-approved`
+label and triggering a merge into the current patch release.
+
+## Searching for Cherry-picks
+
+[A sample search on kubernetes/kubernetes pull requests that are labeled as `cherry-pick-approved`](https://git.k8s.io/kubernetes/pulls?q=is%3Aopen+is%3Apr+label%3Acherry-pick-approved)
+[A sample search on kubernetes/kubernetes pull requests that are labeled as `do-not-merge/cherry-pick-not-approved`](https://github.com/kubernetes/kubernetes/pulls?q=is%3Aopen+is%3Apr+label%3Ado-not-merge%2Fcherry-pick-not-approved)
+
 
-## Cherry Pick Review
+## Troubleshooting Cherry-picks
 
-Cherry pick pull requests are reviewed differently than normal pull requests. In
-particular, they may be self-merged by the release branch owner without fanfare,
-in the case the release branch owner knows the cherry pick was already
-requested - this should not be the norm, but it may happen.
+Contributors may encounter some of the following difficulties when initiating a cherry-pick.
 
-## Searching for Cherry Picks
+- A cherry-pick PR does not apply cleanly against an old release branch.
+In that case, you will need to manually fix conflicts.
 
-See the [cherrypick queue dashboard](http://cherrypick.k8s.io/#/queue) for
-status of PRs labeled as `cherrypick-candidate`.
+- The cherry-pick PR includes code that does not pass CI tests.
+In such a case you will have to fetch the auto-generated branch from your fork, amend the problematic commit and force push to the auto-generated branch.
```

Comment:
You have this statement on a new line, but markdown just joins it to the previous bullet.  Which looks fine to me, just wondering if that was your intent

Reply:
Really!? That's super strange actually I do not see thatat all!?